CHANGELOG.md

```diff
@@ -7,7 +7,15 @@
 ### DEPENDENCIES
 - Bump [ibc-apps/middleware/packet-forward-middleware](https://github.com/cosmos/ibc-apps/tree/main/middleware/packet-forward-middleware) to
     [v8.1.1](https://github.com/cosmos/ibc-apps/releases/tag/middleware%2Fpacket-forward-middleware%2Fv8.1.1)
-    ([\#3533](https://github.com/cosmos/gaia/pull/3533))
+    ([\#3534](https://github.com/cosmos/gaia/pull/3534))
+- Add `v22.2.0` upgrade handler ([\#3538](https://github.com/cosmos/gaia/pull/3538))
```

Comment:
Needs backport to main?

Reply:
done in #3536 which i will merge after the release is done